design/design_template.md

```diff
@@ -0,0 +1,43 @@
+---
+title: feature-name
+target-version: v0.1.2
```

Comment:
Should we make this a placeholder as well? Not that we also end up searching and replacing it in the template every release.